src/Tribe/Cache.php

```diff
@@ -180,6 +181,36 @@ public function delete_expired_transients() {
 				AND a.option_name NOT LIKE '\_transient\_timeout_tribe\_%'
 		";
 		$wpdb->query( $sql );
+
+		if ( wp_using_ext_object_cache() ) {
+			/**
+			 * Toggle trigger to control if WordPress cache should be flushed or not.
+			 *
+			 * When using real object cache we cannot selectively flush a specific cache group.
+			 * This filter allows third-party code to handle the flush request in a more precise way by flushing only
```

Comment:
* :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed
* :no_entry_sign: Line exceeds maximum limit of 100 characters; contains 117 characters

<sub>hash: 3a8a0812848263852d2e4150a5c2916c</sub>